applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -233,18 +271,47 @@ create_conference(Context) ->
     OnSuccess = fun(C) -> on_successful_validation('undefined', C) end,
     cb_context:validate_request_data(<<"conferences">>, Context, OnSuccess).
 
-mute_conference(Context, ConfId) ->
-    Realm = kz_util:get_account_realm(cb_context:account_id(Context)),
-    {'ok', Confs} = get_conferences(Realm, ConfId),
-    Participants = conference_participants(Confs),
-    lager:error("P:~p", [Participants]),
+-spec mute_conference(ne_binary(), ne_binary()) -> kz_json:object().
+mute_conference(Realm, ConfId) ->
+    Participants = conference_participants(get_conference(Realm, ConfId)),
     Conference = kapps_conference:set_id(ConfId, kapps_conference:new()),
     Muted = [
         perform_conference_action(Conference, <<"mute">>, kz_json:get_value(<<"Participant-ID">>, P))
             || P <- Participants, kz_json:is_false(<<"Is-Moderator">>, P), kz_json:is_true(<<"Speak">>, P)
     ],
-    Resp = kz_json:set_value(<<"resp">>, Muted, kz_json:new()),
-    crossbar_doc:handle_json_success(Resp, Context).
+    kz_json:set_value(<<"resp">>, Muted, kz_json:new()).
+
+-spec unmute_conference(ne_binary(), ne_binary()) -> kz_json:object().
+unmute_conference(Realm, ConfId) ->
+    Participants = conference_participants(get_conference(Realm, ConfId)),
+    Conference = kapps_conference:set_id(ConfId, kapps_conference:new()),
+    UnMuted = [
+        perform_conference_action(Conference, <<"unmute">>, kz_json:get_value(<<"Participant-ID">>, P))
+            || P <- Participants, kz_json:is_false(<<"Is-Moderator">>, P), kz_json:is_false(<<"Speak">>, P)
+    ],
+    kz_json:set_value(<<"resp">>, UnMuted, kz_json:new()).
+
+-spec lock_conference(ne_binary()) -> kz_json:object().
+lock_conference(ConfId) ->
+    kapps_conference_command:lock(ConfId),
+    kz_json:set_value(<<"resp">>, <<"ok">>, kz_json:new()).
```

Comment:
Agree on ok payload, will replace it. Could you please elaborate on '_read_only' construction? What is it for, what should it tell the user?